rust/bitcoin_wallet/README.md

```diff
@@ -0,0 +1,127 @@
+# Bitcoin Wallet Example
+
+## Summary
+
+This example dapp shows how to build a basic Bitcoin wallet making use of the
+Internet Computer's [Bitcoin integration](https://smartcontracts.org/docs/developers-guide/concepts/bitcoin-integration.html).
+
+## Step-by-step tutorial
+
+Install the required node modules for the Bitcoin wallet webapp:
+
+```bash
+npm install
+```
+
+While working on the Internet Computer does not require more configuration, working locally does. The additional instructions are provided in [the next section](#testing-locally).
+
+Run the following commands to deploy and initialize the development Internet Identity canister and the Bitcoin wallet canister locally:
+
+```bash
+dfx deploy
+dfx canister call bitcoin_wallet initialize
+```
+
+Run the following commands to deploy and initialize the Bitcoin wallet canister on the Internet Computer:
+
+```bash
+II_CANISTER_ID=identity dfx deploy --network ic bitcoin_wallet_assets
+dfx canister --network ic call bitcoin_wallet initialize
+```
+
+## Testing locally
+
+The Bitcoin wallet invokes the Bitcoin integration API through the management canister.
```

Comment:
This line feels out of place. Should this be moved into a section that explains the code?